File: src/main/java/br/com/leonardoferreira/jirareport/controller/FieldController.java
package br.com.leonardoferreira.jirareport.controller;

import br.com.leonardoferreira.jirareport.domain.vo.JiraField;
import br.com.leonardoferreira.jirareport.service.FieldService;
import java.util.List;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/fields")
public class FieldController {

    @Autowired
    private FieldService fieldService;

    @GetMapping
    public List<JiraField> fields() {
        return fieldService.findAllJiraFields();
    }

}

File: application/src/main/kotlin/br/com/jiratorio/controller/FieldController.kt
package br.com.jiratorio.controller

import br.com.jiratorio.domain.response.FieldResponse
import br.com.jiratorio.service.FieldService
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/fields")
internal class FieldController(
    private val fieldService: FieldService,
) {

    @GetMapping
    fun fields(): List<FieldResponse> =
        fieldService.findAll()

}
